stable/polaris/templates/NOTES.txt

```diff
@@ -0,0 +1,12 @@
+** Please be patient while the chart is being deployed **
+
+Enjoy Polaris and smooth sailing!
+
+{{- if contains "ClusterIP" .Values.dashboard.service.type }}
+
+## To view the dashboard execute these commands:
+
+kubectl port-forward --namespace {{ .Release.Namespace }} svc/{{ include "polaris.fullname" . }}-dashboard 8080:80 &
+open http://localhost:8080
```

Comment:
Is `open` a command in linux, or is that mac-specific?  If it is mac specific, we should probably just make this say "Browse to the place"

Reply:
I think it's a mac command. Changed to a plaintext instruction.